dolphinscheduler-task-plugin/dolphinscheduler-task-mlflow/src/main/java/org/apache/dolphinscheduler/plugin/task/mlflow/MlflowTask.java

```diff
@@ -98,10 +98,15 @@ public void handle() throws Exception {
     @Override
     public void cancelApplication(boolean cancelApplication) throws Exception {
         // cancel process
-        shellCommandExecutor.cancelApplication();
+        if (mlflowParameters.getDeployType().equals(MlflowConstants.MLFLOW_MODELS_DEPLOY_TYPE_DOCKER_COMPOSE)) {
+            String envCommand = mlflowParameters.getDockerComposeEnvCommand();
+            shellCommandExecutor.run(envCommand + "\n" + MlflowConstants.DOCKER_COMPOSE_CANCEL);
```

Comment:
did you task cancel the task locally? I find out your task starts with `docker up -d` and it not alway running task which I think it should not be cancel

Reply:
Is it necessary to implement a cancellation mechanism? I think if it is not necessary, we do have not to cancel the docker container.

Reply:
we should better have a cancel operator here, but IMO, we can only cancel some tasks running for a long time , such as `sleep 200`  in bash which will sleep 200s, and users want to stop them, they could call cancelApp, but you here start the task by fixed `docker compose` command and whether success or fail will exit the command quickly, Am I right? @caishunfeng

Reply:
> we should better have a cancel operator here, but IMO, we can only cancel some tasks running for a long time , such as `sleep 200` in bash which will sleep 200s, and users want to stop them, they could call cancelApp, but you here start the task by fixed `docker compose` command and whether success or fail will exit the command quickly, Am I right? @caishunfeng

Agreed. We should cancel it by using ```docker rm -f``` as discussed in https://github.com/apache/dolphinscheduler/pull/10217/files#r882267962.

Reply:
I think our scenario is just for releasing the model online, and the subsequent operations are not the concern of this component. So, we should not remove the container.